lib/rules/no-duplicate-landmark-labels.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const Rule = require('./base');
+const AstNodeInfo = require('../helpers/ast-node-info');
+
+function createErrorMessage(element) {
+  return `Duplicate label on <${element}> detected. If two of the same landmark elements or landmark roles are found in the same template, ensure that they have a unique label (provided by the aria-label or aria-labelledby attribute).`;
+}
+
+const ROLE_LANDMARK_MAP = {
+  banner: 'header',
+  main: 'main',
+  complementary: 'aside',
+  form: 'form',
+  search: 'form',
+  navigation: 'nav',
+  contentinfo: 'footer',
+};
```

Comment:
Should we include `region: section,`? https://www.w3.org/TR/wai-aria-practices/examples/landmarks/HTML5.html

Reply:
Region is an implicit role that should never be used by authors but let me check on `section` - thank you for sharing the link. It's not the same as the spec so I need to double check a few things.